pmm/aws-staging-start.groovy

```diff
@@ -369,6 +369,7 @@ pipeline {
                                         echo "export PATH=$PWD/pmm2-client-2.0.0/bin:$PATH" >> ~/.bash_profile
                                         source ~/.bash_profile
                                         pmm-admin --version
+                                        cat pmm-agent.log
```

Comment:
This file does not exist at this point.

Reply:
When installation is done via tar or in all cases?

Reply:
I mean, there is nothing before that line that creates this file. pmm-agent itself does not creates it, it logs only to stderr.

Reply:
oh yes, I realize what I did there, pushing fix.